src/server/tsconfig.json

```diff
@@ -8,5 +8,8 @@
       "isolatedModules": false,
       "noEmit": false
     },
+    "ts-node": {
```

Comment:
what is this configuration for?

Reply:
ts-node could not find type for @newrelic/pino-enricher' and could not compile if we don't use it

Reply:
@skarahoda https://stackoverflow.com/a/51666714